tests/functional/timer/test_transfer_create.py

```diff
@@ -61,28 +61,18 @@ def setup_timer_consent_tree_response(identity_id, *data_access_collection_ids):
                         **_dummy_consent_fields,
                     },
                     {
-                        "scope_name": (
-                            "https://auth.globus.org/scopes/"
-                            "actions.globus.org/transfer/transfer"
-                        ),
+                        "scope_name": globus_sdk.TransferClient.scopes.all,
                         "scope": str(uuid.uuid1()),
                         "dependency_path": [100, 101],
                         "id": 101,
                         **_dummy_consent_fields,
                     },
-                    {
-                        "scope_name": globus_sdk.TransferClient.scopes.all,
-                        "scope": str(uuid.uuid1()),
-                        "dependency_path": [100, 101, 102],
-                        "id": 102,
-                        **_dummy_consent_fields,
-                    },
                 ]
                 + [
                     {
                         "scope_name": GCSCollectionScopeBuilder(name).data_access,
                         "scope": str(uuid.uuid1()),
-                        "dependency_path": [100, 101, 102, 1000 + idx],
+                        "dependency_path": [100, 101, 1000 + idx],
```

Comment:
What does this do?

Reply:
To update the transfer create tests to work with the new scope requirements, I had to remove the legacy transfer scope from the consents `RegisteredResponse` and update the consent that used to be dependent on it to not have the old id (102) it its `dependency_path`.